core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -395,17 +419,18 @@ private static String formatFragment(
                 })
                 .collect(toImmutableList());
         builder.append(indentString(1));
+        String hashColumn = partitioningScheme.getHashColumn().map(column -> "[" + column + "]").orElse("");
         if (replicateNullsAndAny) {
             builder.append(format("Output partitioning: %s (replicate nulls and any) [%s]%s\n",
                     partitioningScheme.getPartitioning().getHandle(),
                     Joiner.on(", ").join(arguments),
-                    formatHash(partitioningScheme.getHashColumn())));
```

Comment:
why not keep `using formatHash` method here?

Reply:
We've made `formatHash` to be a non-static method